akka-docs/src/main/paradox/typed/persistence-style.md

```diff
@@ -70,3 +70,14 @@ Java
 :  @@snip [AccountExampleWithNullState.java](/akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/AccountExampleWithMutableState.java) { #account-entity }
 
 @@@
```

Comment:
Shouldn't this `group-java` include the next section as well? Otherwise it will look strange for Scala